client/src/contact/contact.yaml

```diff
@@ -0,0 +1,85 @@
+contact_us_title:
+  en: Contact Us
+  fr: Contactez-nous
+
+contact_us_intro:
+  transform: [markdown]
+  en: |
+    We value your questions and comments. As such, we have provided multiple methods of communication.
+
+
+    If you need to contact us, please email us at questions@tbs-sct.gc.ca.
+  fr: |
+    Vos questions et commentaires nous intéressent. C'est pourquoi nous avons prévu plusieurs méthodes de communication.
+
+
+    Si vous avez besoin de communiquer avec nous, envoyez-nous un courriel à question@tbs-sct.gc.ca.
+
+feedback_title:
+  en: Have Feedback?
+  fr: Avez-vous des commentaires?
+
+feedback_text:
+  transform: [handlebars, markdown]
+  en: |
+    We are always looking for ideas and suggestions to help us improve the current version of the InfoBase. If you would like to share some ideas or comments regarding your experience 
+    with the tool, please take a few minutes to complete our <a href="{{{survey_link}}}" target="_blank" rel="noopener noreferrer">online survey</a>. The feedback you provide will be used
+    to improve the current version of GC InfoBase and prioritize new features.
+  fr: |
+    Nous sommes constamment à la recherche d’idées et de suggestions pour améliorer la version actuelle de l’InfoBase. Le cas échéant, veuillez prendre quelques minutes pour remplir 
+    notre <a href="{{{survey_link}}}" target="_blank" rel="noopener noreferrer">sondage en ligne</a>. En répondant aux quelques questions qui suivent, vous nous aiderez à améliorer
```

Comment:
Super minor, but can shorten these links a bit with a `{{ext_link ... }}` handlebar helper. Fine if you don't, the external link helper is mainly there so we don't have to remember all the extra `rel` attributes every time, being shorter is just a bonus.